core/templates/dev/head/pages/community-dashboard-page/opportunities-list-item/opportunities-list-item.directive.html

```diff
@@ -1,27 +1,33 @@
 <div class="oppia-opportunities-list-item">
   <div class="oppia-opportunities-details">
     <div class="oppia-opportunities-subheading oppia-animate-fadein">
-      <span ng-show="!$ctrl.loadingView"><[$ctrl.opportunity.subheading]></span>
-      <lazy-loading ng-show="$ctrl.loadingView"></lazy-loading>
+      <span ng-show="!$ctrl.loadingViewShow"><[$ctrl.opportunity.subheading | wrapTextWithEllipsis : 25]></span>
```

Comment:
loadingViewIsShown, or as we discussed, flip to fullyLoaded or opportunityDataIsFulllyLoaded

Reply:
Done!